### DISCUSSION TITLE
Kohya HRFix Questions?

### DISCUSSION THREAD

**Discussion by heyitshestia** [lllyasviel/Fooocus#2464] [Q&A] [answered]
Hey! I'm trying to use Kohya for higher resolution images, but the integrated one is so different from the one you can install through extensions making it hard to find any guides that are applicable. Could someone explain to me how to use the integrated kohya?

Also, could someone tell me some good settings for HiresFix? I know my way around it in A1111, but in forge it seems to straight up demolish any generated image no matter the upscaler and/or settings i have been trying to use, and i don't know if its just not working or if i'm doing something wrong...

**Comment by poor7** [comment, marked as answer]
This is not A1111, or Forge. This is Fooocus.

**Reply by heyitshestia** [reply]
Oopsy
